app/src/androidTest/java/net/skyscanner/backpack/bottomsheet/BpkBottomSheetTest.kt

```diff
@@ -58,18 +59,22 @@ class BpkBottomSheetTest : BpkSnapshotTest() {
 
   @Test
   fun default() {
-    snap(setupBottomSheet(), width = 200, height = 200, padding = 0)
+    capture(STATE_COLLAPSED)
```

Comment:
this test was sometimes flaky (also with fb testing before) - always use the convoluted way of testing as the expanded one was consistent before too. will monitor further. if it's still flaky i have another workaround, but that requires a production code change as well, so want to avoid